packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -20,15 +20,52 @@ The Registration Inline Hook is now set up with a status of active.
 
 > **Note:** You can also setup an inline hook using an API. See [Inline Hooks Management API](/docs/reference/api/inline-hooks/#create-inline-hook) for further information.
 
+### Enabling the Registration Inline Hook
+
+The procedure to enable the Registration Inline Hook is dependent on the type of org you are using: Okta Identity Engine or Okta Classic. Follow the procedure for your specific org.
+
+After you enable the Registration Hook, you are ready to preview and test the example.
```

Comment:
Yes, I guess I was trying to "single-source" this rather than put it at the end of each procedure. But that's not too excessive. I'll move it. Thanks.